contracts/upgradeable_contracts/erc20_to_native/HomeBridgeErcToNative.sol

```diff
@@ -15,6 +15,12 @@ contract HomeBridgeErcToNative is EternalStorage, BasicBridge, BasicHomeBridge {
         require(msg.value > 0);
         require(msg.data.length == 0);
         require(withinLimit(msg.value));
+        address blockRewardAddress = blockRewardContract();
+        if(blockRewardAddress.call(abi.encodeWithSignature("mintedTotallyByBridge(address)",  address(this)))) {
```

Comment:
Seems that such kind of the check is not actual. Since the corresponding changes are in `master` (https://github.com/poanetwork/poa-network-consensus-contracts/pull/201) and already applied to the Dai chain. Let's assume that the method exists.